msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java

```diff
@@ -189,6 +191,13 @@ public void getAccount(@NonNull final String identifier,
     private void getAccountInternal(@NonNull final String identifier,
                                     @NonNull final GetAccountCallback callback,
                                     @NonNull final String publicApiId) {
+        try {
+            validateNonNullArg(identifier, "identifier");
+            validateNonNullArg(identifier, "callback");
```

Comment:
Should be `callback` as first param

Reply:
That probably won't make a difference if the `callback` is `null`. Need to throw `IllegalStateException` instead for `callback` validation